linera-core/src/client/mod.rs

```diff
@@ -422,6 +423,58 @@ where
     }
 }
 
+// Tracks which chain_id<>block_height pairs have been successfully processed.
```

Comment:
(Technically, we only download them to process them later.)

Reply:
Done in [c660103](https://github.com/linera-io/linera-protocol/pull/2666/commits/c66010359a10c4462def29e143edf267aec539dc).